core/src/test/java/com/test/ExternalSecurityMemberAccessTest.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package com.test;
+
+import com.opensymphony.xwork2.ognl.SecurityMemberAccessTest;
+
+/**
+ * Runs the same test suite using a SecurityMemberAccess class that is outside the
+ * com.opensymphony.xwork2.ognl package.
+ */
+public class ExternalSecurityMemberAccessTest extends SecurityMemberAccessTest {
+
+    @Override
+    protected void assignNewSma(boolean allowStaticFieldAccess) {
+        sma = new ExternalSecurityMemberAccess(allowStaticFieldAccess);
```

Comment:
> We have two SecurityMemberAccessTest in two different packages to test two different behaviours. One test is in the package com.opensymphony.xwork2.ognl which is excluded.

@lukaszlenart Do you think this is an okay solution?

Reply:
Nope :) Yet testing the `MemberAccess` behaviour for different settings makes sense, this is the last defense line :)